async-nats/src/lib.rs

```diff
@@ -778,6 +778,23 @@ impl Subscriber {
             receiver,
         }
     }
+
+    /// Unsubscribes from subscription, draining all remaining messages.
+    ///
+    /// # Examples
+    /// ```
+    /// # #[tokio::main]
+    /// # async fn unsubscribe() -> Result<(), Box<dyn std::error::Error>> {
+    /// let mut client = async_nats::connect("demo.nats.io").await?;
+    ///
+    /// let mut subscriber = client.subscribe("foo".into()).await?;
+    ///
+    ///  subscriber.unsubscribe();
+    /// # Ok(())
+    /// # }
+    pub fn unsubscribe(mut self) {
```

Comment:
I'd prefer to just call this close

```suggestion
    pub fn close(mut self) {
```

Reply:
I prefer to stick with NATS names here. After all we're unsubscribing, not closing the subscription.